### PR TITLE
NAS-107304 / 20.10 / NAS-107304

### PR DESCRIPTION
**Pull request by dszidi**
Fixed the following
- Sidenav background color and hover styles 
- Combobox
- radio buttons
- ipwithnetmask
- entity-form-list
- buttons styles
- consolidated/ renamed some global CSS files
- Various CSS tweaks

### CODE REVIEW

File: src/app/pages/common/entity/entity-form/entity-form.component.html [hunk 1]
```diff
@@ -33,15 +33,15 @@ <h4 *ngIf="fieldSet.label && isFieldsetAvailabel(fieldSet)" class="fieldset-labe
                   ix-auto-type="button"
                   ix-auto-identifier="{{saveSubmitText | uppercase}}"
                   >{{saveSubmitText | translate}}</button>
-                <button id="goback_button" *ngIf="conf.route_success" class="btn  btn-block btn-lg mat-accent" type="button" (click)="goBack()" mat-button color="accent"
+                <button id="goback_button" *ngIf="conf.route_success" class="btn  btn-block btn-lg"9 type="button" (click)="goBack()" mat-button color="default"
```
Comment (william-gr) [inline src/app/pages/common/entity/entity-form/entity-form.component.html:36]: "9 is it a typo?

File: src/app/services/theme/theme.service.ts [hunk 2]
```diff
@@ -10,12 +10,12 @@ export const DefaultTheme = {
       name:'ix-dark',
       label: "iX Dark",
       labelSwatch:"blue",
-      description:'FreeNAS 11.2 default theme',
+      description:'FreeNAS 12 default theme',
```
Comment (william-gr) [inline src/app/services/theme/theme.service.ts:13]: Might as well call it TrueNAS now

File: src/app/pages/common/entity/entity-form/entity-form.component.html (hunk 1 above)
Comment (dszidi) [reply src/app/pages/common/entity/entity-form/entity-form.component.html:36]: Tmux strikes again. Will fix

File: src/app/services/theme/theme.service.ts (hunk 2 above)
Comment (dszidi) [reply src/app/services/theme/theme.service.ts]: Done

**Pull request merged**